src/main/java/com/tobmistaketracker/detector/MistakeDetectors/VerzikMeleeChancedTracker.java

```diff
@@ -0,0 +1,76 @@
+package com.tobmistaketracker.detector.MistakeDetectors;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Actor;
+import net.runelite.api.NPC;
+import net.runelite.api.Player;
+import net.runelite.api.coords.WorldArea;
+import net.runelite.api.events.AnimationChanged;
+import net.runelite.api.gameval.AnimationID;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Slf4j
+public class VerzikMeleeChancedTracker {
+    @Getter
+    private final HashSet<Player> playersThatChancedMelee = new HashSet<>();
+
+    private Player lastTickTarget;
+    private WorldArea lastTickTargetArea;
+    private WorldArea lastTickVerzikArea;
+
+    private static final Set<Integer> VERZIK_ATTACK_ANIMATIONS = Set.of(
+            AnimationID.VERZIK_PHASE3_ATTACK_MELEE,
+            AnimationID.VERZIK_PHASE3_ATTACK_MAGIC,
+            AnimationID.VERZIK_PHASE3_ATTACK_RANGED
+    );
+
+    public void setVerzikAttackInfo(NPC verzik){
+        if (verzik == null)
+        {
+            emptyTickData();
+        } else {
+            Actor target = verzik.getInteracting();
+            if (target instanceof Player)
+            {
+                lastTickTarget = (Player) target;
+                lastTickTargetArea = lastTickTarget.getWorldArea();
+                lastTickVerzikArea = verzik.getWorldArea();
+            }
+            else
+            {
+                emptyTickData();
+            }
+        }
+    }
+
+    public void checkPlayerWronglyTanked(AnimationChanged event)
+    {
+        int animationId = event.getActor().getAnimation();
+
+        if (VERZIK_ATTACK_ANIMATIONS.contains(animationId))
+        {
+            if (lastTickTarget != null && lastTickTargetArea != null && lastTickVerzikArea != null)
+            {
+                if (isWronglyTanked(lastTickVerzikArea, lastTickTargetArea))
+                {
+                    playersThatChancedMelee.add(lastTickTarget);
+                }
```

Comment:
Are we trying to add a mistake for all members of the party that were in melee range? It's technically not their mistake though, it's just the tanker's mistake. I don't think they should get penalized, especially since it could have caused them to miss ticks. What are your thoughts?